modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -6,33 +6,17 @@ To use Prometheus to collect, store, and query metrics about the {devworkspace}
 
 .Prerequisites
 
-* The link:https://github.com/devfile/devworkspace-operator/blob/v0.10.0/deploy/deployment/kubernetes/objects/devworkspace-controller-metrics.Service.yaml[`devworkspace-controller-metrics` Service] is exposing metrics on port `8443`.
+* The `devworkspace-controller-metrics` Service is exposing metrics on port `8443`. This is pre-configured by default.
```

Comment:
I recommend dropping the dash because that's the mainstream variant according to https://www.merriam-webster.com/dictionary/pre-configure:
```suggestion
* The `devworkspace-controller-metrics` Service is exposing metrics on port `8443`. This is preconfigured by default.
```